db/db_impl_open.cc

```diff
@@ -412,6 +432,16 @@ Status DBImpl::Recover(
     default_cf_handle_ = new ColumnFamilyHandleImpl(
         versions_->GetColumnFamilySet()->GetDefault(), this, &mutex_);
     default_cf_internal_stats_ = default_cf_handle_->cfd()->internal_stats();
+    // When recovering from a DB which already contains persistent stats CF,
+    // the CF is already created in VersionSet::ApplyOneVersionEdit, but
+    // column family handle was not. Need to explicitly create handle here.
+    if (mutable_db_options_.stats_persist_period_sec != 0 &&
+        persist_stats_cf_handle_ == nullptr) {
```

Comment:
If we reach here, is it possible that `mutable_db_options_.stats_persist_period_sec != 0 && persist_stats_cf_handle_ != nullptr` ?

Reply:
Yes, when we are recovering from a RocksDB instance that perviously had persistent_stats column family created, we would need to skip the `CreateColumnFamily` call above as `ApplyOneVersionEdit` would have already created the column family. However, the column family handle still needs to be explicitly created here